rasr/crp.py

```diff
@@ -60,6 +61,41 @@ def __repr__(self):
     def html(self):
         return object_to_html(self.__dict__)
 
+    def set_executables(self, rasr_root, rasr_arch="linux-x86_64-standard"):
+        """
+        Set all executables to a single RASR path
+
+        Is (not yet) compatible with tk.Path inputs
+
+        :param str rasr_root: director path of a RASR repository
```

Comment:
```suggestion
        :param str rasr_root: directory path of a RASR repository
```
typo

Or maybe even better docstring
```suggestion
        :param str rasr_root: path of a directory containing the RASR repository
```